public/components/content-list-drawer/content-list-drawer.html

```diff
@@ -334,6 +334,52 @@
                         </li>
                     </ul>
                 </div>
+                
+                <!--Charts ATOM DRAWER-->
```

Comment:
Since most of this template is the same as the story-questions maybe we could duplicate less code by wrapping the individual parts that are different into ng-ifs? If we could move the `getAtomDisplayName` somewhere where we can share it with this editor then the Content type could come from this function and we'd only have the last list in this template that is different between the two atom types.

Reply:
Updated the code to make it more general